test/toplevel/test_virtualprocess.jl

```diff
@@ -86,6 +86,19 @@ end
     end
 end
 
+@testset "test to_simple_module_usages" begin
+    @test JET.to_simple_module_usages(:(using A, B; export a, b)) ==
+        Expr[quote using A, B end, quote export a, b end]
```

Comment:
```suggestion
    @test JET.to_simple_module_usages(:(using A, B)) ==
        Expr[:(using A), :(using B)]
    @test JET.to_simple_module_usages(:(export a, b)) ==
        Expr[:(export a), :(export b)]
```
`to_simple_module_usages` isn't supposed to take `:block` expression.

Reply:
The comment in `to_simple_module_usages` 

```
    if length(x.args) != 1
        # using A, B, export a, b
        return Expr.(x.head, x.args)
    else
```

suggested something like that?

Reply:
To elaborate `# using A, B, export a, b`, this code is supposed to handle patterns like `using A, B` or `export a, b`.

Reply:
Thanks!